eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -8,21 +8,27 @@
 import logging
 from rest_framework.exceptions import APIException
 from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.db import models
 from enrollment import api
 from enrollment.errors import CourseModeNotFoundError
 from enrollment.errors import CourseEnrollmentExistsError
 from course_modes.models import CourseMode
 from opaque_keys.edx.keys import CourseKey
 from student.models import CourseEnrollment
+from edxfuture import get_program
+import pprint
```

Comment:
Just marking so we dont forget to remove the pprint before merging

Reply:
Ok